data/xml/Q17.xml

```diff
@@ -406,71 +465,90 @@
+    <url>https://www.aclweb.org/anthology/Q17-1033</url>
+    <video href="https://vimeo.com/285803652" tag="video" />
   </paper>
 
   <paper id="1034">
     <title>Phrase Table Induction Using In-Domain Monolingual Data for Domain Adaptation in Statistical Machine Translation</title>
     <author><first>Benjamin</first><last>Marie</last></author>
     <author><first>Atsushi</first><last>Fujita</last></author>
     <year>2017</year>
-    <abstract>We present a new framework to induce an in-domain phrase table from in-domain monolingual data that can be used to adapt a general-domain statistical machine translation system to the targeted domain.Our method first compiles sets of phrases in source and target languages separately and generates candidate phrase pairs by taking the Cartesian product of the two phrase sets.  It then computes inexpensive features for each candidate phrase pair and filters them using a supervised classifier in order to induce an in-domain phrase table. We experimented on the language pair English-French, both translation directions, in two domains and obtained consistently better results than a strong baseline system that uses an in-domain bilingual lexicon. We also conducted an error analysis that showed the induced phrase tables proposed useful translations, especially for words and phrases unseen in the parallel data used to train the general-domain baseline system.</abstract>
-    <href>https://transacl.org/ojs/index.php/tacl/article/download/1166/270</href>
+    <month>December</month>
+    <doi>10.1162/tacl_a_00075</doi>
+    <abstract>We present a new framework to induce an in-domain phrase table from in-domain monolingual data that can be used to adapt a general-domain statistical machine translation system to the targeted domain. Our method first compiles sets of phrases in source and target languages separately and generates candidate phrase pairs by taking the Cartesian product of the two phrase sets. It then computes inexpensive features for each candidate phrase pair and filters them using a supervised classifier in order to induce an in-domain phrase table. We experimented on the language pair English–French, both translation directions, in two domains and obtained consistently better results than a strong baseline system that uses an in-domain bilingual lexicon. We also conducted an error analysis that showed the induced phrase tables proposed useful translations, especially for words and phrases unseen in the parallel data used to train the general-domain baseline system.</abstract>
     <pages>487–500</pages>
-    <url>http://www.aclweb.org/anthology/Q17-1034</url>
+    <url>https://www.aclweb.org/anthology/Q17-1034</url>
   </paper>
 
   <paper id="1035">
     <title>Joint Prediction of Word Alignment with Alignment Types</title>
-    <author><first>Anahita</first><last>Mansouri Bigvand</last></author>
+    <author><first>Anahita Mansouri</first><last>Bigvand</last></author>
```

Comment:
I think this is incorrect: see [her LinkedIn page, for example](https://www.linkedin.com/in/anahita-mansouri-03788416/?originalSubdomain=ca). I assume this comes straight from the XML they gave you?

Reply:
Yup.

```xml
<string-name name-style="western"><given-names>Anahita Mansouri</given-names><x xml:space="preserve"> </x><surname>Bigvand</surname></string-name>
```

Reply:
Better just to manually correct it here in the PR. I’m also putting this in a mail to the MIT Press folks since the error is on their side.

Reply:
done